storage/rocksdb/ha_rocksdb.cc

```diff
@@ -19020,6 +19028,28 @@ static bool parse_fault_injection_params(
   return false;
 }
 
+static bool rocksdb_ddse_dict_init(
+    dict_init_mode_t dict_init_mode, uint,
+    [[maybe_unused]] List<const dd::Object_table> *tables,
+    List<const Plugin_tablespace> *tablespaces) {
+  // TODO: Remove assert(false); once we add the full rocksdb dd support
+  assert(false);
+
+  assert(tables);
+  assert(tables->is_empty());
+  assert(tablespaces);
+  assert(tablespaces->is_empty());
+
+  assert(dict_init_mode == DICT_INIT_CREATE_FILES ||
+         dict_init_mode == DICT_INIT_CHECK_FILES);
+
+  static Plugin_tablespace dd_space(rocksdb_dd_space_name, "", "", "",
+                                    rocksdb_hton_name);
+  tablespaces->push_back(&dd_space);
```

Comment:
On a second thought, tablespaces are purely InnoDB concept. So nothing should be added here?

Reply:
I think DD expect mysql namespace always exists.. Let me find related code in a while..

Reply:
The current DD layer assumes it does, but I am patching it out in my prototype.
I am concerned about pretending something exists when MyRocks does not even support the concept of it.